sbi/inference/posteriors/vi_posterior.py

```diff
@@ -1,69 +1,566 @@
 # This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
```

Comment:
Please add the `VIPosterior` to the documentation [here](https://github.com/mackelab/sbi/blob/main/docs/docs/reference.md)